src/Php81/Resources/stubs/FiberExit.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * Exception thrown when destroying a fiber. This exception cannot be caught by user code.
+ */
+final class FiberExit extends Exception
```

Comment:
This class doesn't actually exist. It's an internal marker class that is not (or should not be) visible from user code.